### DISCUSSION TITLE
Pole figure in m.u.d

### DISCUSSION THREAD

**Discussion by prastyani28** [mtex-toolbox/mtex#1422] [Ask Anything] [answered]
Hi all,

I have a question about the pole figure plotting in MTEX.
I plotted my pole figure by using the script:

```
plotPDF(ori,h,ori.CS,'contourf','minmax','all')
mtexColorbar
```

And I got this pole figure plot:
![300R2V1_Site 5 Map 5_pole figure after MAD](https://user-images.githubusercontent.com/67193615/166632534-50e93615-3781-4c83-9bef-da65255f7613.jpg)

What is the unit used in this color bar? Is it m.u.d?
If it is not m.u.d (multiple uniform distributions), how do I make it to m.u.d? 

Any help would be very much appreciated!

I'm still new in learning MTEX, so I need more guidance and assistance :)

Best,
Erina

**Comment by lfgmorales** [comment, marked as answer]
Hi Erina,

My understanding is that if you didnt calculate an odf, as shown in your script, the scale is not in multiples of uniform distribution, so I would do that before plotting the pole figures. Also, it looks like you only have a couple of grains of this phase, and therefore calculating an odf or imposing contours like you did is not completely meaningful in terms of statistics. In this case I would rather plot the orientantions as single points.

Cheers

Luiz

Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 4 May 2022, at 08:34, NanaAcademics ***@***.***> wrote:

﻿

Hi all,

I have a question about the pole figure plotting in MTEX.
I plotted my pole figure by using the script:

plotPDF(ori,h,ori.CS,'contourf','minmax','all')
mtexColorbar


And I got this pole figure plot:
[300R2V1_Site 5 Map 5_pole figure after MAD]<https://user-images.githubusercontent.com/67193615/166632534-50e93615-3781-4c83-9bef-da65255f7613.jpg>

What is the unit used in this color bar? Is it m.u.d?
If it is not m.u.d (multiple uniform distributions), how do I make it to m.u.d?

Any help would be very much appreciated!

I'm still new in learning MTEX, so I need more guidance and assistance :)

Best,
Erina

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1422>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN76LENFKNX47HCWZMDVIILARANCNFSM5VA7DW5A>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1422", "url": "https://github.com/mtex-toolbox/mtex/discussions/1422", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Reply by prastyani28** [reply]
Hi Luiz,

thanks for your quick reply!

Yeah, there are only 30 grains for this plot. What do you mean by plotting orientation as a single point? 
Is it also by using pole figure? or other function in MTEX?

Looking forward for more suggestions for you :)

Best,
Erina

**Comment by lfgmorales** [comment]
Hi Erina,


If you remove the option 'contourf' of your plot, you will have a scatter plot, which shows the orientations of those 30 grains without the contours, I think this is more meaningful. Also, because it is not clear from your code what ori means (if they are mean orientations of "grains", or all the EBSD points of a certain phase, or something else), just make sure that ori is also something meaningful. From the pole figures you sent, it looks like you have like 10 grains or so (and not 30), is it possible?


cheers



Luiz





Luiz F. G. Morales
ScopeM & Structural Geology and Tectonics Group
ETH Zürich
HPM C57.3

Otto-Stern-Weg 3

8093 Zürich

Switzerland

Phone: + 41 44 633 3746

<tel:+41446333746>E-mail: ***@***.***



"No trees were destroyed in the sending of this message, however, a significant number of electrons were terribly inconvenienced"


________________________________

**Reply by prastyani28** [reply]
Hi Luiz,

thanks again for your reply!

I defined 'ori' as:

```
ori = ebsd('Quartz-new').orientations;
```
I got the information about the number of quartz grains by writing:
```
grains(ebsd('Quartz-new')))
```

And I got these:
```
Phase  Grains  Pixels     Mineral  Symmetry  Crystal reference frame
     0      31     302  notIndexed                                   
     1      30    2092  Quartz-new      -3m1       X||a*, Y||b, Z||c*
```

Does it mean that it has 30 grains with 2092 orientations of quartz? 
Please let me know if I have a wrong understanding of this.

And I tried to calculate the ODF first (as you have suggested) so that I get m.u.d of my pole figure, but now I don't know how to change my plot to scatter plot. I tried to remove the 'contourf' but it still gives me the same plot with contour.

This is the code that I used:
```
odf_quartz=calcDensity(ebsd('Quartz-new').orientations);
h=[Miller(1,0,0,odf_quartz.CS),Miller(1,1,0,odf_quartz.CS),...
    Miller(0,0,1,odf_quartz.CS)];
plotPDF(odf_quartz,h,'minmax')
```

This is the result that I got:
![ask](https://user-images.githubusercontent.com/67193615/166653449-43b1a9f8-5d4d-44e5-8174-374d92935958.jpg)

Looking forward to your answer :)

Best,
Erina

**Reply by prastyani28** [reply]
Sorry! I've just realized that if I use scatter plot I do not need to know the m.u.d 😅 

But then my pole figure plot becomes like this:
![ask](https://user-images.githubusercontent.com/67193615/166654669-ae59cb7a-2c1a-491f-8307-0affb1491027.jpg)

Is it still OK to be considered a pole figure plot?

Best,
Erina

**Comment by lfgmorales** [comment]
Hi Erina,


In this case, I would calculate grains first, and after that plot the meanOrientation of each grain


something like


grains=calcGrains(ebsd,'angle',10*degree)

plotPDF(grains,grains.meanOrientation,h)


cheers



Luiz



ScopeM & Structural Geology and Tectonics Group

ETH Zürich
HPM C57.3

Otto-Stern-Weg 3

8093 Zürich

Switzerland

Phone: + 41 44 633 3746

<tel:+41446333746>E-mail: ***@***.***



"No trees were destroyed in the sending of this message, however, a significant number of electrons were terribly inconvenienced"


________________________________

**Reply by prastyani28** [reply]
OK, thanks! 😄
